packages/proto/src/messages/ibc/ibcMsgTransfer.ts

```diff
@@ -0,0 +1,44 @@
+import { Coin } from '../../proto/cosmos/base/coin'
+import { Height } from '../../proto/cosmos-ibc/ibc/core/client'
+import { MsgTransfer } from '../../proto/cosmos-ibc/ibc/applications/tx'
+
+export function createIBCMsgTransfer(
```

Comment:
memo is missing

Reply:
We should create a separate PR for this, since we would need to add it to `transactions` and `eip712` as well.

Reply:
I want to minimize the scope of this PR as much as possible, so we can merge it in and unblock other work. I can open PRs for these fixes as soon as we finalize this refactor.

Reply:
They are different. The memo I'm referring to goes in the IBC packet, not in the transaction.

Reply:
I understand, but this would break the code within the `transactions` package that calls the `proto` implementation. Furthermore, the `transactions` package wouldn't support receiving the IBC `memo` field as a parameter until we add it, and the `EIP-712` package would not support it either.

In order to fully support this change, we have to make changes across `transactions` and `eip712`. That should be done separately.

Reply:
ok, your plan sounds good to me. Can you create a ticket for it to add it to v12?